aws/terminator.yml

```diff
@@ -39,100 +50,100 @@
-        virtualenv_python: python3.9
+        virtualenv_python: "{{ lambda_python_version }}"
+      check_mode: false
+
     - name: package terminator requirements
       tags: lambda
-      lambda_package:
-        src: "{{ packaging_dir }}/terminator-requirements"
+      archive:
+        format: zip
         dest: "{{ packaging_dir }}/terminator-requirements.zip"
-        include:
-          - "{{ packaging_dir }}/terminator-requirements/python/lib/python3.9/site-packages/*"
-        exclude:
-          # pre-compiled bytecode
+        path:
+          - "{{ packaging_dir }}/terminator-requirements/python/lib/{{ lambda_python_version }}/site-packages/*"
+        exclusion_patterns:
           - "*.pyc"
-          # packaging information not needed at runtime
           - "*.dist-info/*"
-          # only used for botocore documentation generation
-          - "{{ packaging_dir }}/terminator-requirements/python/lib/python3.9/site-packages/docutils/*"
-          # installed during creation of the virtualenv
-          - "{{ packaging_dir }}/terminator-requirements/python/lib/python3.9/site-packages/pip/*"
-          - "{{ packaging_dir }}/terminator-requirements/python/lib/python3.9/site-packages/wheel/*"
-          - "{{ packaging_dir }}/terminator-requirements/python/lib/python3.9/site-packages/setuptools/*"
-          - "{{ packaging_dir }}/terminator-requirements/python/lib/python3.9/site-packages/pkg_resources/*"
-          - "{{ packaging_dir }}/terminator-requirements/python/lib/python3.9/site-packages/easy_install.py"
+          - "{{ packaging_dir }}/terminator-requirements/python/lib/{{ lambda_python_version }}/site-packages/docutils/*"
+          - "{{ packaging_dir }}/terminator-requirements/python/lib/{{ lambda_python_version }}/site-packages/pip/*"
+          - "{{ packaging_dir }}/terminator-requirements/python/lib/{{ lambda_python_version }}/site-packages/wheel/*"
+          - "{{ packaging_dir }}/terminator-requirements/python/lib/{{ lambda_python_version }}/site-packages/setuptools/*"
+          - "{{ packaging_dir }}/terminator-requirements/python/lib/{{ lambda_python_version }}/site-packages/pkg_resources/*"
+          - "{{ packaging_dir }}/terminator-requirements/python/lib/{{ lambda_python_version }}/site-packages/easy_install.py"
+
     - name: publish terminator requirements layer
       tags: lambda
       lambda_layer:
-        name: "{{ api_name }}-terminator-requirements"
-        description: "Python requirements for {{ api_name }}-terminator"
+        name: "{{ lambda_layer_name }}"
+        description: "Python requirements for {{ lambda_function_name }}"
         compatible_runtimes:
-          - python3.9
-        path: "{{ packaging_dir }}/terminator-requirements.zip"
+          - "{{ lambda_python_version }}"
+        content:
+          zip_file: "{{ packaging_dir }}/terminator-requirements.zip"
         license_info: GPL-3.0-only
-        region: "{{ aws_region }}"
```

Comment:
Don't we still need the region? Can we add this to `module_defaults`?

Reply:
updated